_includes/payment-state.md

```diff
@@ -0,0 +1,31 @@
+{% assign api_resource = include.api_resource | default: "`paymentorder`" %}
+{% assign transaction = include.transaction | default: "`authorization`" %}
+
+## Payment State
+
+The `state` field on the {{ transaction }} does not indicate whether a given
```

Comment:
You can wrap variables in whatever Markdown formatting you'd like, as the variables are processed by Liquid before any Markdown formatting is applied. Once the Markdown processor sees the content, all Liquid statements will have been processed and gone.

```suggestion
The `state` field on the `{{ transaction }}` does not indicate whether a given
```